optuna/pruners/hyperband.py

```diff
@@ -1,14 +1,14 @@
-import optuna
+import math
+from typing import List  # NOQA
```

Comment:
We shouldn't need `# NOQA` as the symbol is always used (not just a type string).
```suggestion
from typing import List
```